ERCS/erc-0000.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Ethereum Interoperability Layer Wallet Usage
+description: A standard interface for wallets and SDKs to perform token transfers and cross-chain calls via the Ethereum Interoperability Layer (EIL).
+author: Alex Forshtat (@forshtat), Dror Tirosh (drortirosh), Shahaf Nacson (shahafn), Yoav Weiss (@yoavw), Marissa Posner (@marissaposner)
+discussions-to: TBD
```

Comment:
Please create a discussions topic in Eth Magicians with a link to this PR https://ethereum-magicians.org/c/ercs/57